packages/opentelemetry-instrumentation-bedrock/opentelemetry/instrumentation/bedrock/__init__.py

```diff
@@ -216,8 +216,18 @@ def _set_anthropic_completion_span_attributes(span, request_body, response_body)
     )
 
     if Config.enrich_token_usage:
-        prompt_tokens = _count_anthropic_tokens([request_body.get("prompt")])
```

Comment:
The reason we put this under `if Config.enrich_token_usage` is because `_count_anthropic_tokens` is expensive to run and we want to give users an option to disable it. If you're getting the data from the request - no need to put it under this `if`